nservicebus/definition-event.include.md

```diff
@@ -0,0 +1 @@
+An [**event**](/nservicebus/messaging/messages-events-commands.md#event) is a message that is used to communicate that some action has taken place. It is published from a single logical sender, and is processed by (potentially) many receivers.
```

Comment:
>It is published from a single logical sender

events can be published from receivers too.  It'd be better to say single logical endpoint instead of saying a sender or a receiver.